Makefile

```diff
@@ -49,28 +50,22 @@ test/unit:
 	@go test -v -tags=unit -coverpkg ./... -coverprofile cover-unit.coverprofile -covermode=count ./pkg/...
 
 .PHONY: test/e2e
-test/e2e: cluster/prepare
-	@echo Running tests:
-	@touch deploy/empty-init.yaml
-	# This is not recommended way or running the tests (see https://github.com/operator-framework/operator-sdk/blob/master/doc/test-framework/writing-e2e-tests.md#running-go-test-directly-not-recommended)
-	# However, this way we will have a consistent way of running tests on Travis and locally. The downside
-	# is that Operator testing harness downloads things manually using `go mod` when executing the tests.
-	# Here is a corresponding Operator SDK call:
-	# operator-sdk test  local --go-test-flags "-tags=integration -coverpkg ./... -coverprofile cover-e2e.coverprofile -covermode=count" --namespace ${NAMESPACE} --up-local --debug --verbose ./test/e2e
-	go test -tags=integration -coverpkg ./... -coverprofile cover-e2e.coverprofile -covermode=count -mod=vendor ./test/e2e/... -root=$(PWD) -kubeconfig=$(HOME)/.kube/config -globalMan deploy/empty-init.yaml -namespacedMan deploy/empty-init.yaml -test.v -singleNamespace -parallel=1 -localOperator -test.timeout 0
+test/e2e: setup/operator-sdk
+	@echo Running e2e local tests:
+	operator-sdk test local --go-test-flags "-tags=integration -coverpkg ./... -coverprofile cover-e2e.coverprofile -covermode=count -timeout 0" --operator-namespace $(NAMESPACE) --up-local --debug --verbose ./test/e2e
```

Comment:
Unfortunately, there is no way to run this from `go test`. There's no switch to tell the testing harness to use a specific namespace. Those options are only exposed in `operator-sdk test` command.